docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+Working with Lights
+###################
+
+:summary: This tutorial demonstrates the creation and manipulation of LightSetups in Habitat-sim.
+
+.. contents::
+    :class: m-block m-default
+
+A **LightSetup** consists of a set of LightInfo structures defining the common configuration
+of a set of point lights used to render objects in a scene. Once defined and registered,
+a LightSetup can be assigned to any subset of objects in the scene, including the scene asset itself.
+
+Each **LightInfo** structure in a LightSetup defines the `color`, `position`, and `LightPositionModel` of a single point light source.
+The LightPositionModel defines the coordinate frame of the light and can be set to any of:
+
+.. code:: python
+
+    habitat_sim.gfx.LightPositionModel.CAMERA
+    habitat_sim.gfx.LightPositionModel.GLOBAL
+    habitat_sim.gfx.LightPositionModel.OBJECT
+
+Each LightSetup is registered in the simulator via a unique key. Two default lighting setups are pre-defined:
+
+
+- **habitat_sim.gfx.DEFAULT_LIGHTING_KEY**
+
+  8 white, GLOBAL lights at scene bounding box corners.
+
+- **habitat_sim.gfx.NO_LIGHT_KEY**
+
+  0 lights (for Flat shaded assets).
+
+
+Additional custom setups can be created and registered via the **simulator.set_light_setup** function:
+
+.. code:: python
+
+    sim.set_light_setup(new_light_setup, "my_custom_lighting_key")
+
+Any existing LightSetup can be queried with **simulator.get_light_setup**:
+
+.. code:: python
+
+    custom_light_setup = sim.set_light_setup(n"my_custom_lighting_key")
+
+
+`Working with Lights Example:`_
+===============================
+
+The example code below demonstrates the usage of default and custom LightSetups and is runnable via:
+
+.. code:: shell-session
+
+    $ python examples/tutorials/lighting_tutorial.py
+
+First, we import modules we will need, define some convenience functions, and initialize the Simulator, Agent, and objects.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [setup]
+    :end-before: # [/setup]
+
+`Scene Lighting`_
+=================
+
+By default, the scene will be shaded with no lights using the **habitat_sim.gfx.NO_LIGHT_KEY** LightSetup.
+This configuration is ideal for scene assets with illumination baked into textures (e.g. building scans such as MP3D).
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [default scene lighting]
+    :end-before: # [scene swap shader]
+
+.. image:: images/lighting-setups-images/0.jpg
+    :width: 45%
+
+To use a custom light setup for the scene, edit the **habitat_sim.SimulatorConfiguration.scene_light_setup**
```

Comment:
This is great! This right here is a use-case. 

Could you expand on this and say why someone might want to use a different light-setup? (Presumably because not all 3D scans have lighting baked in; anything else?)